src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_VOLUME = 10;
+
+    private final Object[] values;
+    private final Object[] keys;
+    private int countObject = 0;
+
+    public StorageImpl() {
+        values = new Object[MAX_VOLUME];
+        keys = new Object[MAX_VOLUME];
+    }
+
     @Override
     public void put(K key, V value) {
+        int index = indexOf(key);
+        if (index < 0) {
+            values[countObject] = value;
+            keys[countObject] = key;
+            countObject++;
+            return;
+        }
+        values[index] = value;
+        keys[index] = key;
     }
 
     @Override
     public V get(K key) {
-        return null;
+        int index = indexOf(key);
+        return index < 0 ? null : (V) values[index];
```

Comment:
```suggestion
        return index == NOT_FOUND ? null : (V) values[index];
```
`private static final int NOT_FOUND = -1`
for better readibility

Reply:
ok